.github/workflows/unit-tests-pytest.yml

```diff
@@ -14,69 +14,53 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
-    - name: Setup Miniconda
-      uses: conda-incubator/setup-miniconda@v2
+    - name: Create Python ${{ matrix.python-version }} environment
+      uses: mamba-org/provision-with-micromamba@main
```

Comment:
Does upstream recommend using "master"? Consider using a major version instead of "master" for long term stability.

Reply:
According to https://github.com/mamba-org/provision-with-micromamba/releases, it is `v12`.  I'll try that.